src/login/71-seat.rules.in

```diff
@@ -37,7 +37,13 @@ SUBSYSTEM=="usb", ATTR{bDeviceClass}=="00", TAG+="seat"
 # Allow USB hubs (and all downstream ports) to be assigned to a seat
 SUBSYSTEM=="usb", ATTR{bDeviceClass}=="09", TAG+="seat"
 
-# 'Plugable' USB hub, sound, network, graphics adapter
+# 'HP Multiseat T100' USB hub, sound, network, graphics adapter
+SUBSYSTEM=="usb", ATTR{idVendor}=="03f0", ATTR{idProduct}=="0032", ENV{ID_AUTOSEAT}="0"
+
+# 'Plugable UD-PRO8' USB hub, sound, network, graphics adapter
+SUBSYSTEM=="usb", ATTR{idVendor}=="1a40", ATTR{idProduct}=="0201", ENV{ID_AUTOSEAT}="0"
```

Comment:
hmm, so iirc 2230 is plugable's USB vendor ID, and 1a40 is "Terminus", i.e. the vendor of the usb hub chip? Isn't there any better way to detect the plugable device, i.e. whether the usb chip is actually built into a Plugable device? If you do "lsusb -v", isn't there any field that would make this better recognizable?

Reply:
If there is something here, I don't see it.  The output is quite large, here's the two devices:

https://pastebin.com/tFnNGDGq

Bus 001 Device 003: ID 17e9:0377  
Bus 001 Device 002: ID 1a40:0201  
Bus 001 Device 001: ID 1d6b:0002  

From the host USB controller(1d6b:0002) the USB hub in the Plugable (1a40:0201) is the primary device.  This is different from the HP device, where the HP device itself is the primary device.  The Plugable device is 17e9:0377.  I uploaded a screenshot below which makes this easier to visualize.